chrome/content/zotero/preferences/preferences_cite.js

```diff
@@ -159,5 +160,45 @@ Zotero_Preferences.Cite = {
 	 **/
 	styleImportError: function () {
 		alert(Zotero.getString('styles.installError', "This"));
+	},
+	
+	/*
+	 * Builds the bibliographyLocale drop-down from available CSL locales
+	 */
+	populateBibliographyLocaleList: function () {
+		// Initialize default format drop-down
+		var bibliographyLocale = '';
+		bibliographyLocale = Zotero.Prefs.get("export.bibliographyLocale");
+		
+		if(!bibliographyLocale) {
+			bibliographyLocale = Zotero.locale;
+		}
+		if(!bibliographyLocale) {
+			bibliographyLocale = 'en-US';
+		}
+		
+		var locales = null;
+		var localesLocation = "resource://zotero/schema/locales.json"; 
+		
+		locales = JSON.parse(Zotero.File.getContentsFromURL(localesLocation));
```

Comment:
IMO loading available locales from file should be moved to Zotero.Styles.init and supported locales available via some property of Zotero.Styles.